community/modules/remote-desktop/chrome-remote-desktop/outputs.tf

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+output "startup_script" {
+  description = "script to load and run all runners, as a string value."
+  value       = module.client_startup_script.startup_script
+}
+
+output "name" {
+  description = "Name of any instance created"
+  value       = module.instances[*].name
+}
```

Comment:
Returning a conditional value based on instance count for >1 could work, but the wait module doesn't accept null or "" as valid inputs for `instance_name` parameter, which is mandatory.  For example:

```
value = length(module.instances.instance_count) > 0 ? "instance-name-0" : null
```

If I return a bogus string instead of null eg. value = "blah", the wait module can't detect the end of the start up script (as expected) and will continue until timing out.

So, a roundabout way to ask - how can we approach this to handle instance_count =0 scenarios? (I can remove wait module but that doesn't seem right too)

Reply:
I do not see that as an actual problem. 

I think that you would only use the wait module if you were creating an instance. Otherwise, you would either not wait, or wait on the instance that runs this script. In the case of a slurm partition, one would not call wait.

Reply:
Just tested deploying this with `instance_count: 0` and **without the wait module** and can confirm it working as expected - seeing the Nvidia and CRD ansible playbooks for being pushed to GCS bucket.

If `instance_count: 0` is used **with the wait module**,  terraform is unable to deploy the blueprint as it is missing the mandatory name parameter

```
module.wait.data.google_compute_instance.vm_instance: Reading...
╷
│ Error: must provide either `self_link` or `name`
│ 
│   with module.wait.data.google_compute_instance.vm_instance,
│   on modules/wait-for-startup/main.tf line 20, in data "google_compute_instance" "vm_instance":
│   20: data "google_compute_instance" "vm_instance" {
│ 
```

I'll close this issue as this is expected behaviour, given you wouldn't use wait module without creating an instance anyway (but can still use the same module for script only)

@nick-stroud FYI